elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/AggregationDataStoreIntegrationTest.java

```diff
@@ -259,6 +239,7 @@ public void whereFilterTest() throws Exception {
 
     @Test
     @Disabled
+    //FIXME
```

Comment:
This comment should explain what features need to be supported to make this test pass.